content/config/daemon/proxy.md

```diff
@@ -1,44 +1,28 @@
 ---
-description: Learn about controlling and configuring the Docker daemon using systemd
-keywords: dockerd, daemon, systemd, configuration, proxy, networking
-title: Configure the daemon with systemd
+description: Learn how to configure the Docker daemon to use an HTTP proxy
+keywords: dockerd, daemon, configuration, proxy, networking, http_proxy, https_proxy, no_proxy, systemd, environment variables
+title: Configure the daemon to use a proxy
 aliases:
   - /articles/host_integration/
   - /articles/systemd/
   - /engine/admin/systemd/
```

Comment:
Looks like this page moved, so we need a another redirect from `/config/daemon/systemd/`



```suggestion
  - /articles/systemd/
  - /config/daemon/systemd/
  - /engine/admin/systemd/
```